<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SomePlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuLibrary">
    <property name="title">
     <string>Library</string>
    </property>
    <addaction name="actionManageLibrary"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSetTimer"/>
    <addaction name="actionEqualizer"/>
   </widget>
   <widget class="QMenu" name="menuPlayer">
    <property name="title">
     <string>Player</string>
    </property>
    <addaction name="actionSavePlaylist"/>
    <addaction name="actionImport"/>
    <addaction name="actionAddStream"/>
   </widget>
   <widget class="QMenu" name="menuHelp_2">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOnlineHelp"/>
   </widget>
   <addaction name="menuHelp"/>
   <addaction name="menuPlayer"/>
   <addaction name="menuLibrary"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp_2"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionManageLibrary">
   <property name="text">
    <string>Manage library</string>
   </property>
  </action>
  <action name="actionEqualizer">
   <property name="text">
    <string>Equalizer</string>
   </property>
  </action>
  <action name="actionSetTimer">
   <property name="text">
    <string>Set timer</string>
   </property>
  </action>
  <action name="actionSavePlaylist">
   <property name="text">
    <string>Save playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOnlineHelp">
   <property name="text">
    <string>Online help</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import playlists</string>
   </property>
  </action>
  <action name="actionAddStream">
   <property name="text">
    <string>Add stream</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
